source/common/router/config_utility.cc

```diff
@@ -63,5 +63,17 @@ Http::Code ConfigUtility::parseRedirectResponseCode(
   }
 }
 
+Http::Code ConfigUtility::parseClusterNotFoundResponseCode(
+    const envoy::api::v2::RouteAction::ClusterNotFoundResponseCode& code) {
+  switch (code) {
+  case envoy::api::v2::RouteAction::SERVICE_UNAVAILABLE:
+    return Http::Code::ServiceUnavailable;
+  case envoy::api::v2::RouteAction::NOT_FOUND:
```

Comment:
May be add a comment that this is the default currently.

Reply:
I don't see how the default value is relevant to this code.

Also, default setting value is documented in the proto definition.

Reply:
I agree that commenting on the default here is not needed.